rust/phoenix-channel/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "phoenix-channel"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tokio-tungstenite = { version = "0.19.0", features = ["native-tls-vendored"] } # Only native-tls compiles to musl.
```

Comment:
Sorry, just noticed this, rustls is the one that works well with musl in all enviroments, you should use `rustls-tls-native-roots`

Reply:
I do remember trying this but it didn't work for some reason. Note that I have a CI check for compiling to musl!

I'll try again and report back :)

Reply:
Changed to rustls now, not sure why that didn't work earlier.